### PR TITLE
Adding Kixer ad network to amp-ad

### LINKED ISSUE #5689: Intent To Implement: Kixer ad network for amp-ads
# Overview
Adding support for the [Kixer](http://kixer.com) Ad Network to `<amp-ad>` elements.

## Comments 
Kixer will submit a PR for a `kixer/amp-ad` branch from Kixer's `kixer-com/amphtml` fork.

### PR DESCRIPTION
**Pull request by jvelis**
Closes #5689

### CODE REVIEW

**Comment by lannka** [comment]
@jvelis could you try to implement our render-start API? See [developer guidelines](https://github.com/ampproject/amphtml/blob/master/ads/README.md#developer-guidelines-for-a-pull-request)

**Comment by jvelis** [comment]
Hi @lannka! just added render-start. Thanks!

**Comment by lannka** [comment]
sorry I should have mentioned together, there's also the no-content API you can call in case no ad is returned. Do you want to add that as well?

**Comment by jvelis** [comment]
Hey @lannka, Kixer is a Direct Response ad network, so we offer a 100% fill. Should we still include the `no-content-available` API?
Thanks

File: examples/ads.amp.html [hunk 1]
```diff
@@ -111,7 +111,7 @@
     <a href="#imobile">I-Mobile</a> |
     <a href="#industrybrains">Industrybrains</a> |
     <a href="#inmobi">InMobi</a> |
-    <a href="#kargo">Kargo</a> |
+    <a href="#kixer">Kixer</a> |
```
Comment (lannka) [inline examples/ads.amp.html]: could you add kargo back?

File: ads/kixer.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ * limitations under the License.
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+
+/* global
+__kxamp: false,
+__kx_ad_slots: false,
+__kx_ad_start: false,
+*/
```
Comment (lannka) [inline ads/kixer.js]: what is this for?

File: examples/ads.amp.html (hunk 1 above)
Comment (jvelis) [inline examples/ads.amp.html]: Added Kargo link back. Thanks.

File: ads/kixer.js (hunk 2 above)
Comment (jvelis) [inline ads/kixer.js]: This prevents the "var is not defined" lint error on those three variables.

**Pull request merged**